### ISSUE TITLE
Multiple errors on capacitor electron app

### ISSUE THREAD

**Issue by IsaacSomething** [ionic-team/capacitor#888]
**Info**

Ionic:

   ionic (Ionic CLI)                          : 4.2.1
   Ionic Framework                        : @ionic/angular 4.0.0-beta.12
   @angular-devkit/build-angular : 0.7.5
   @angular-devkit/schematics     : 0.7.5
   @angular/cli                               : 6.1.5
   @ionic/angular-toolkit               : 1.0.0

System:

   NodeJS : v8.12.0 (C:\Program Files\nodejs\node.exe)
   npm    : 5.10.0
   OS     : Windows 10     

     No @ionic/core ToastController detected, please make sure you have included the web components.
     electron\node_modules\@capacitor\electron\dist\electron-bridge.js:sourcemap:17
     VM95:1 GET file:///D:/runtime.js net::ERR_FILE_NOT_FOUND
     VM95:1 GET file:///D:/polyfills.js net::ERR_FILE_NOT_FOUND
     VM95:1 GET file:///D:/styles.js net::ERR_FILE_NOT_FOUND
     VM95:1 GET file:///D:/vendor.js net::ERR_FILE_NOT_FOUND
     VM95:1 GET file:///D:/main.js net::ERR_FILE_NOT_FOUND

This is a fresh install following the online documentation.
There is also an error in the console.

     Cannot read property 'Splashscreen' of undefined

**Comment by bandodebits** [comment]
Same error!

**Comment by bandodebits** [comment]
Any fix?

**Comment by ezraroda** [comment]
Same here

**Comment by zsoflin** [comment]
This worked for me. Not sure of the implications... 
in index.html, replace`<base href="/">` with `<base href="./">`.
https://github.com/electron/electron/issues/1769#issuecomment-257424968

**Comment by webprofusion-chrisc** [comment]
Is this problem windows only?

Adding `./` to the base href worked slightly as per the suggestion by  @zsoflin 

However proceeding results in several other issues related to the `file://` protocol being used to load CSS etc:
![image](https://user-images.githubusercontent.com/2445502/48878766-d8a96600-ee43-11e8-94f8-e071600c9a05.png)

Given that capacitor with Electron will be a pretty important part of Ionic desktop apps in the Enterprise (i.e big user base, no mac os)  it would be great to get this resolved so we can start developing properly.

This has been like this for a while and I understand the team has lots of other priorities, but I also know Max has been working on windows and enterprise topics recently. Pinging @mlynch for an opinion/prioritisation update.

Basically using Capacitor with Electron on Windows 10 as per the getting started guide (https://capacitor.ionicframework.com/docs/electron/) does not work out the box currently, I'm assuming it does work for Mac OS?

**Comment by zsoflin** [comment]
I updated to the latest versions of capacitor and electron, and this resolved the Fetch API errors:
`cd electron`
`npm install @capacitor/electron@1.0.0-beta.11 --save`
`npm install electron@3.0.10 --save-dev`

**Comment by webprofusion-chrisc** [comment]
@zsoflin thanks, so did I just now. I still get this error but so far I assume it's not serious: 
`C:\Work\Misc\certify\electron\node_modules\@capacitor\electron\dist\electron-bridge.js:17 No @ionic/core ToastController detected, please make sure you have included the web components.`

**Comment by zsoflin** [comment]
That is an issue I still have. I haven't found a resolution for it yet.

**Comment by ajayambre** [comment]
Had this same issue. 
Followed @zsoflin instructions and resolved this with below steps.

Replace `<base href="/">` with `<base href="./">` in `index.html`
`cd electron`
`npm install @capacitor/electron@1.0.0-beta.11 --save`
`npm install electron@3.0.10 --save-dev`

This error still remains but the app works fine.
`@ionic/core ToastController detected, please make sure you have included the web components.`

**Comment by mlynch** [comment]
So there are like 3-4 issues I'm seeing in here. What is the one biggest one that we can focus on?

**Comment by jcesarmobile** [comment]
All the issues should be solved on latest beta, except for the base href one as that is angular related, and I don’t think we can solve it in Capacitor itself, instead whe used from ionic CLI it could be automated, and if not using ionic then users should specify the base href on the build command themselves before doing the copy.
As far as I know, other frameworks will also face similar issues, in example in vue apps users will have to configure a similar property in a json file. Again, we can’t do anything from Capacitor itself, but we can do it from ionic CLI if we integrate the capacitor build

**Comment by lincolnthree** [comment]
The base href issue seems to be an initial blocker for even the newest of new users, so I'd say that's probably a good one to start with. It will fail for anyone who tries to follow the getting-started instructions on the capacitor website. I ended up here and found the solution, anyway, but still :) Anyway...Might be worth a note at least.

**Comment by NiklasMerz** [comment]
see #639 for the base href issues.

**Comment by mlynch** [comment]
Closing as the `<base>` issue is a duplicate of #639 and looks like the other issues have been resolved.

**Comment by claraneta** [comment]
I already changed the index.html yet it's still not working. 

![electron_error](https://user-images.githubusercontent.com/50764260/63426087-99488e00-c444-11e9-83c8-3ce15a3690c6.png)

**Comment by claraneta** [comment]
> Had this same issue.
> Followed @zsoflin instructions and resolved this with below steps.
> 
> Replace `<base href="/">` with `<base href="./">` in `index.html`
> `cd electron`
> `npm install @capacitor/electron@1.0.0-beta.11 --save`
> `npm install electron@3.0.10 --save-dev`
> 
> This error still remains but the app works fine.
> `@ionic/core ToastController detected, please make sure you have included the web components.`

it does not still works on me . Can you suggest something ?
